src/mechanics/timeIntegration/ExplicitRungeKutta.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <vector>
+
+namespace NuTo
+{
+namespace TimeIntegration
+{
+template <typename state>
+class ExplicitRungeKutta
+{
+
+public:
+    //! @brief Initialization with method specific parameters (butcher tableau)
+    ExplicitRungeKutta(std::vector<std::vector<double>> aa, std::vector<double> bb, std::vector<double> cc)
+    {
+        for (auto elm : aa)
+        {
+            a.push_back(elm);
+        }
+        for (auto elm : bb)
+        {
+            b.push_back(elm);
+        }
+        for (auto elm : cc)
+        {
+            c.push_back(elm);
+        }
+    }
+
+    //! @brief Performs one RungeKutta step
+    template <typename F>
+    state doStep(F f, state w0, double t0, double h)
```

Comment:
Can you provide a brief documentation here? Esp. how `f` has to look like. 

I am still struggeling with the name, `DoStep` would be our convention. But, I guess, you want to have a similar naming as the `boost::odeint` stuff. Still, not sure if we should stick to our naming convention.

Reply:
I would also stick to our naming convention. In case use the boost interface, there could be a similar method in the header that calls our method. This should be optimized by the compiler.